README.md

```diff
@@ -33,7 +33,7 @@ Use `riviere` if you want an easy way to log all the HTTP traffic for your serve
     12. [health](#options_health)
     13. [outbound.enabled](#options_outbound_enabled)
     14. [outbound.maxBodyValueChars](#options_outbound_max_body_value_chars)
-    15. [outbound.blacklistedPaths](#options_outbound_blacklisted_paths)
+    15. [outbound.blacklistedPathRegex](#options_outbound_blacklisted_path_regex)
```

Comment:
🔧  This is an backwards incompatible change that will require a major version bump. Have you checked whether we use the blacklistedPaths in projects?